test/infrastructure/docker/Makefile

```
@@ -91,7 +91,7 @@ test-e2e: ## Run the end-to-end tests
 
 E2E_CONF_FILE ?= e2e/local-e2e.conf
 SKIP_RESOURCE_CLEANUP ?= false
-FOCUS ?= Basic
+FOCUS ?= Basic|Full
 run-e2e:
 	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=20m -tags=e2e -e2e.config="$(abspath $(E2E_CONF_FILE))" -skip-resource-cleanup=$(SKIP_RESOURCE_CLEANUP) -ginkgo.focus='$(FOCUS)'
```

Might as well just remove the focus here if we want to run all tests
```suggestion
run-e2e:
	go test ./e2e -v -ginkgo.v -ginkgo.trace -count=1 -timeout=20m -tags=e2e -e2e.config="$(abspath $(E2E_CONF_FILE))" -skip-resource-cleanup=$(SKIP_RESOURCE_CLEANUP)'
```